lib/pure/pathnorm.nim

```diff
@@ -100,3 +100,4 @@ proc normalizePath*(path: string; dirSep = DirSep): string =
   result = newStringOfCap(path.len)
   var state = 0
   addNormalizePath(path, result, state, dirSep)
+  if result == "" and path != "": result = "."
```

Comment:
`addNormalizePath` is exported, too. But you exclude your fix from that function, why?

Reply:
fixed